src/models/update_org_request.rs

```diff
@@ -46,7 +47,8 @@ impl UpdateOrgRequest {
             autojoin_by_domain: None,
             restrict_to_domain: None,
             legacy_org_id: None,
-            // require_2fa_by: None,
+            require_2fa_by: None,
+            extra_domains: None,
```

Comment:
```suggestion
            extra_domains: Vec::new(),
```

Reply:
Actually, a correction on this one...
we should instead define `extra_domains` as
```
#[serde(rename = "extra_domains", skip_serializing_if = "Option::is_none")]
pub extra_domains: Option<Vec<String>>,
```
to distinguish between the cases where there's no change vs setting to an empty array. Then this line stays
```
extra_domains: None,
```